sparse/coo/umath.py

```diff
@@ -507,8 +507,8 @@ def _check_broadcast(self):
         )
 
         if full_shape != non_ndarray_shape:
-            raise ValueError('All ndarrays must be broadcastable to the shape without ndarrays {}'
-                             .format(non_ndarray_shape))
+            raise ValueError('Please make sure that the broadcast shape of just the sparse arrays is '
```

Comment:
So in the case of a binary operation between dense and sparse as in the issue, "just the sparse arrays" is the one array, so broadcasting the sparse array is not allowed?

Reply:
Correct.